ska_conda/pkg_defs/mica/meta.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: mica
+  version:  {{ GIT_DESCRIBE_TAG }}
+
+build:
+  noarch: python
+  script_env:
+    - SKA_TOP_SRC_DIR
+
+source:
+  path: {{ SKA_TOP_SRC_DIR }}/mica
+
+
+# the build and runtime requirements. Dependencies of these requirements
+# are included automatically.
+requirements:
+  # Packages required to build the package. python and numpy must be
+  # listed explicitly if they are required.
+  build:
+    - python
+    - setuptools
+    - testr
+  # Packages required to run the package. These are the dependencies that
+  # will be installed automatically whenever the package is installed.
+  run:
+    - python
+    - six
+    - setuptools
+    - matplotlib
+    - numpy
+    - pytest
+    - django
```

Comment:
???

Reply:
For a bunch of packages, we have imports that are only used in certain circumstances.  For mica, django is needed for mica.web.  Obviously mica.web is useless unless we're running kadi webserver, in which case there is already a django.

I agree it is probably fine to remove django from here.